platform/security/idp/security-idp-server/encryption.properties

```diff
@@ -0,0 +1,7 @@
+#Property descriptions at http://ws.apache.org/wss4j/config.html
```

Comment:
Why is this file at the root of the idp-server module? Same question about the keystore and other properties file.